apps/src/publicKeyCryptography/NumberedSteps.jsx

```diff
@@ -17,15 +17,120 @@ export default function NumberedSteps(props) {
   return (
     <table>
       <tbody>
-        {React.Children.map(props.children, (child, index) => (
-          <tr>
-            <td style={style.td}>{(index + 1) + ')'}</td>
-            <td style={style.td}>{child}</td>
-          </tr>
-        ))}
+        {React.Children.map(props.children, (child, index) => React.cloneElement(child, {index}))}
```

Comment:
By themselves, the children don't know their index within the parent.  The [`React.cloneElement`](https://facebook.github.io/react/docs/top-level-api.html#react.cloneelement) call within the map operation applies the correct index to each child before rendering them.